CONTRIBUTING.md

```diff
@@ -112,7 +112,7 @@ When you're finished with the changes, create a pull request, also known as a PR
 - Fill the "Ready for review" template so that we can review your PR. This template helps reviewers understand your changes as well as the purpose of your pull request. 
 - Don't forget to [link PR to issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) if you are solving one.
 - Enable the checkbox to [allow maintainer edits](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) so the branch can be updated for a merge.
-Once you submit your PR, your proposal wil be review. We may ask questions or request for additional information.
+Once you submit your PR, your proposal will be review. We may ask questions or request for additional information.
```

Comment:
There is another file called `contributing.mdx`. Can you also make these changes there?

`/docs/pages/docs/about/contributing.mdx`

Reply:
ok, I am on it

Reply:
Done